blockchain/chain_sync/src/sync.rs

```diff
@@ -346,18 +346,18 @@ where
     }
     /// Validates message root from header matches message root generated from the
     /// bls and secp messages contained in the passed in block and stores them in a key-value store
-    fn validate_msg_meta(&self, block: &Block) -> Result<(), Error> {
+    async fn validate_msg_meta(&self, block: &Block) -> Result<(), Error> {
```

Comment:
there is nothing async in this function?

Reply:
Probably a remnant from an old refactor